### ISSUE TITLE
[Question] Uploading file other then image files (e.g. text files)

### ISSUE THREAD

**Issue by anwar3606** [hinesboy/mavonEditor#705]
Does mavon editor provide any functionality to upload files other images?

I tried to use `imageFilter` function but it doesn't working even if I return true. 

Any guide?

**Comment by jiawulin001** [comment]
Hi, please provide more details, eg. a reproduction repo will be better, Thx!

**Comment by anwar3606** [comment]
Simple, I just want to upload `pdf` or `text` file and put their link inside markdown. Just like image.

**Comment by jiawulin001** [comment]
`markdown` cannot preview non-image type files, so there is no way to upload other format files, we must know the address of the file, for example：
![image](https://user-images.githubusercontent.com/14244944/128655512-2e3a8588-cd52-4d19-aafe-09d34f8c8e20.png)

I think this feature is similar to uploading images and should be a nice new feature.

**Comment by anwar3606** [comment]
Agree, GitHub and Gitlab has similar feature in their markdown editors.

**Comment by tomaszmac** [comment]
> `markdown` cannot preview non-image type files, so there is no way to upload other format files, we must know the address of the file, for example： ![image](https://user-images.githubusercontent.com/14244944/128655512-2e3a8588-cd52-4d19-aafe-09d34f8c8e20.png)
> 
> I think this feature is similar to uploading images and should be a nice new feature.

Did you find a way to upload files like pdf or txt?

**Comment by anwar3606** [comment]
> > `markdown` cannot preview non-image type files, so there is no way to upload other format files, we must know the address of the file, for example： ![image](https://user-images.githubusercontent.com/14244944/128655512-2e3a8588-cd52-4d19-aafe-09d34f8c8e20.png)
> > I think this feature is similar to uploading images and should be a nice new feature.
> 
> Did you find a way to upload files like pdf or txt?

Right now I'm using another component to upload file and manually changing the markdown text by myself

**Comment by tomaszmac** [comment]
> > > `markdown` cannot preview non-image type files, so there is no way to upload other format files, we must know the address of the file, for example： ![image](https://user-images.githubusercontent.com/14244944/128655512-2e3a8588-cd52-4d19-aafe-09d34f8c8e20.png)
> > > I think this feature is similar to uploading images and should be a nice new feature.
> > 
> > 
> > Did you find a way to upload files like pdf or txt?
> 
> Right now I'm using another component to upload file and manually changing the markdown text by myself

Could you tell me how you did it? I'm still struggling with it.
